apps/web/src/stories/footer.stories.tsx

```diff
@@ -1,9 +1,18 @@
+import React from 'react';
 import { Footer } from '@/components/footer';
 import { Meta, StoryObj } from '@storybook/react';
+import { ApplicationMetadataContext } from '@/components/application-metadata-provider';
 
 const meta: Meta<typeof Footer> = {
   title: 'Footer',
   component: Footer,
+  decorators: [
+    (Story) => (
+      <ApplicationMetadataContext.Provider value={{ data: { version: '1.0.0', versionUpdatedAt: '2023-01-01' } }}>
```

Comment:
Just wondering, hard coded version & date? I expect the version to match the release number?

Reply:
Guess this is related to `Laatste live-versie v1.0.0` task above?

Reply:
So basically, the expected version number in the footer is the latest semantic versioning. This PR will update the current 1.8.0 (2023-03-03) release, isn't it? So the 1.8.1 or 1.9.0 wil be the one expected to be shown in the footer.

Reply:
No this is nit related to the task. The version number here is regarding the story, stories will usually have mocked values. The actual component will indeed have the actual version.